src/views/InvolvementPanel/InvolvementPanel.js

```diff
@@ -18,6 +19,10 @@ export default class InvolvementPanel extends React.Component {
         text: 'Volunteer',
         view: <VolunteerFormView />
       },
+      TEACHER: {
```

Comment:
Oh, interesting. I wasn't aware. I unfortunately don't have a platform with which to test Internet Explorer. Would you mind debugging and figuring out what's causing that?